src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs

```diff
@@ -416,6 +416,8 @@ private bool HasJobAttribute(CustomAttributeData attributeData)
                         
             descr.Parameters = parameters;
             descr.TriggerParameterDescriptor = parameters.OfType<TriggerParameterDescriptor>().FirstOrDefault();
+            descr.FilterAttributes = method.GetCustomAttributes().OfType<IFunctionInvocationFilter>();
```

Comment:
>descr.FilterAttributes =  [](start = 12, length = 25)

Can you move these to the FromMethod() helper above? That's called by this func.